SPECS/libcap/libcap.spec

```diff
@@ -17,6 +17,7 @@ into a set of distinct privileges.
 Summary:        Development files for libcap
 Group:          Development/Libraries
 Requires:       %{name} = %{version}-%{release}
+Provides:       pkgconfig(libcap)
```

Comment:
Explicit provides should be versioned -- `pkgconfig(libcap) = %{version}-%{release}`